zh-Hans.json

```diff
@@ -683,6 +683,23 @@
 	"dpa.changed.text2": "自从您最后一次同意。请同意DPA以继续。",
 	"dpa.required": "请阅读并接受数据处理附录。",
 	"dpa.text": "数据处理附录（DPA）",
+	"edgeHostname.advancedSettings.hint.DeliverByHttps": "是否通过DNS记录声明服务端支持的HTTP协议。使用该功能，客户端可以在DNS域名解析阶段完成HTTP协议协商，提升性能。\n注：当访客分区规则中配置了重定向到某个IP地址，则调度域名解析出的HTTPS记录，不能代表重定向的目标地址是否支持某个HTTP协议",
```

Comment:
@jungho73 Will the "\n" will be interpreted as a line break? Otherwise, we shall remove it.

Reply:
@Jianghai-Quantil 
Sorry, I didn't notice the newline. If we need a newline, I'll have to change the way I show hints. It would be better to reflect the newline, right?

Reply:
Yes, it is nice to have. We can start a new task to support that. Don't have to do it now.